test/srt/test_input_embeddings.py

```diff
@@ -0,0 +1,149 @@
+import json
+import os
+import subprocess
+import tempfile
+
+import torch
```

Comment:
This looks good! Can you make it a unit test like the following examples
https://github.com/sgl-project/sglang/blob/main/test/srt/test_srt_endpoint.py
https://github.com/sgl-project/sglang/blob/main/test/srt/models/test_generation_models.py